packages/common/src/hooks/useDrawer.ts

```diff
@@ -2,20 +2,32 @@ import create from 'zustand';
 import LocalStorage from '../localStorage/LocalStorage';
 
 type DrawerController = {
+  hoverActive: Record<string, boolean>;
```

Comment:
a little clumsy, but doesn't need interacting with so I thought I could get away with it?